less/elements.less

```
@@ -1,3 +1,33 @@
+/* ---------------------------------- */
+/*  Copyable Text                     */
+/* ---------------------------------- */
+
+copyable-text {
+  text-decoration: underline;
+  text-decoration-style: dashed;
+  cursor: pointer;
+  &:hover, &:has(:focus-visible) {
```

Is `:focus-within` equivalent?

```suggestion
  &:hover, &:focus-within {
```

---

`:focus-within` in this case behaves more like `:focus`, so it will stay highlighted after clicking.